src/main/java/no/nav/tag/dittNavArbeidsgiver/services/aareg/AAregService.java

```diff
@@ -39,10 +39,7 @@ public OversiktOverArbeidsForhold hentArbeidsforhold(String orgnr,String juridis
                 throw new RuntimeException(message);
             }
             return respons.getBody();
```

Comment:
kan dette kaste nullpointer exception også?

Reply:
ser det blir håntert i aaregcontroller

Reply:
blir det ikke kastet to errors nå, dersom aareg feiler? 1 i kontrolleren og 1 i servicen?

Reply:
jo det er det, fjernet det som er i servicen nå